ONSdigital/ras-frontstage
# DAC report - Error handing and surfacing

# What and why?
This PR addresses the issues raised by the DAC testing since some of the error handling in frontstage does not follow the GOV.UK guidelines. This adds an `Error: ` tag in the page title so screen readers notify of an error immediately. It also adds a hidden `Error` tag for screen readers in the page content.
# How to test?
Make sure that on the required pages seen on the trello card the Error: label comes up in the page title and either using a screen reader or inspecting the page source verify that there is a hidden `Error` tag in the title where the error happened.

<img width="663" alt="image" src="https://user-images.githubusercontent.com/41422777/203978925-8c6924dd-5728-4bff-8378-9bf95d3ed63d.png">

# Trello
https://trello.com/c/8OxuuUlV/2073-dac-report-error-handling-usability
